modules/web/middleware/binding.go

```diff
@@ -42,7 +42,7 @@ func AssignForm(form interface{}, data map[string]interface{}) {
 		if fieldName == "-" {
 			continue
 		} else if len(fieldName) == 0 {
-			fieldName = util.ToSnakeCase(field.Name)
+			fieldName = field.Name
```

Comment:
Maybe the snake_case is done by purpose:

* snake_case: for auto filled form, matching the form element's  `name`
* PascalCase:  for non-form variables

?

If  the form-variables  and  non-form-variables  are mixed together, it  would  be very difficult to  understand what a variable is  used for, and may cause uncontrollable overwriting?

Reply:
@wxiaoguang 
Thanks for your review.
You are right, it is difficult to distinguish and may cause uncontrollable overwriting.

How about the following changes.

For non-form variables:
```
// in func
ctx.Data["HelloData"]
// in template
{{.HelloData}}
```
For auto filled form:
```
// in func
ctx.Data["Form"] = forms.FormStruct {
    Variable: vaule,
    // ....
}
// in template
{{.Form.Variable}}
```

Reply:
I am not sure and I haven't thought about it carefully.

Maybe the Technical Oversight Committee could help.

Reply:
Personally, I like the idea `{{.Form.Variable}}`, it's more readable and go-style. And there will be a clear way to check if it's an error page to repost with entered form: `{{if .Form}} /* keep the values in .Form*/ {{end}}`

Reply:
I see the point of the snake_case but using the `{{.Form}}` would prevent us from having to do that.

Reply:
So there is one potential issue with the `.Form` object if you're using go code.

I'm not certain if there's any code that does this - but I could imagine some code that doesn't have the type of the Form object but would have been able to update the values in the `ctx.Data["some_field"]` which by convention was a string.

Reply:
In some pages, there are several forms in one page. Need to consider a new solution to solve this problem.